test/benchmark/bm_iroha_ed25519.cpp

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <ed25519/ed25519.h>
+
+#include <cstdlib>
+#include <vector>
+
+#include <benchmark/benchmark.h>
+
+auto ConstructRandomVector(int64_t size) {
+  std::vector<unsigned char> v;
+  v.reserve(static_cast<int>(size));
+  for (int i = 0; i < size; ++i) {
+    v.push_back(static_cast<unsigned char>(std::rand() % size));
```

Comment:
Shouldn't we call `srand` before somewhere?

Reply:
https://en.cppreference.com/w/cpp/numeric/random/rand

>If rand() is used before any calls to srand(), rand() behaves as if it was seeded with srand(1). 

This behavior should be fine here.